### PR TITLE
update amqp stubs from "pdezwart/php-amqp"

### PR DESCRIPTION
**Pull request by wenpeng**

### CODE REVIEW

**Comment by isfedorov** [comment]
Hello @wenpeng,
Thank you for your contribution, but it's required to perform some steps before your PR can be approved:
1)Please revert all code-style related changes and leave only notable ones (please see https://github.com/JetBrains/phpstorm-stubs/blob/master/CONTRIBUTING.md for additional notes)
2)Please note that after your changes tests started to fail. Please update your changes so that tests pass again to be able to merge your PR

File: amqp/amqp.php [hunk 1]
```diff
@@ -121,13 +121,22 @@
 /**
  *
  */
-define('AMQP_OS_SOCKET_TIMEOUT_ERRNO', 536870923);
+define('AMQP_OS_SOCKET_TIMEOUT_ERRNO', 536870947);
```
Comment (isfedorov) [inline amqp/amqp.php]: One of tests fails due to this incorrect value. Please see https://github.com/pdezwart/php-amqp/blob/master/php_amqp.h for details on how this constant is calculated

File: amqp/amqp.php [hunk 2]
```diff
@@ -1199,30 +1419,27 @@ public function setName($exchange_name) { }
...
-     * Declare a new exchange on the broker.
-     * @return int
-     * @throws AMQPExchangeException
-     * @throws AMQPChannelException
-     * @throws AMQPConnectionException
-     */
-    public function declare(){}
+    public function getConnection()
+    {
+    }
```
Comment (isfedorov) [inline amqp/amqp.php]: Method `declare()` should not be removed.

File: amqp/amqp.php [hunk 3]
```diff
@@ -1282,12 +1503,14 @@ public function bind($exchange_name, $routing_key = null, array $arguments = arr
      *                             and it equals to latest consumer_tag on queue,
      *                             it will be interpreted as latest consumer_tag usage.
      *
-     * @throws AMQPChannelException    If the channel is not open.
+     * @return bool;
```
Comment (isfedorov) [inline amqp/amqp.php]: type hint should not end with ;

**Comment by wenpeng** [comment]
@isfedorov Hi, Thank you for your review, it's has been updated.

**Pull request merged**